test/party_client.py

```diff
@@ -434,3 +434,31 @@ def get_pending_surveys_with_originator_party_id(self, party_id, expected_status
     def delete_pending_surveys_with_batch_no(self, batch_no, expected_status=202):
         response = self.client.delete(f"/party-api/v1/pending-surveys/{batch_no}", headers=self.auth_headers)
         self.assertStatus(response, expected_status)
+
+    def get_password_verification_token(self, party_id, expected_status=200):
+        response = self.client.get(f"/party-api/v1/respondents/{party_id}/password-verification-token")
+        self.assertStatus(response, expected_status)
+        return response.json
+
+    def post_password_verification_token(self, party_id, payload, expected_status=200):
+        response = self.client.post(
+            f"/party-api/v1/respondents/{party_id}/password-verification-token", json=payload, headers=self.auth_headers
+        )
+        self.assertStatus(response, expected_status, response.json)
+
+    def delete_password_verification_token(self, party_id, token):
+        response = self.client.delete(
+            f"/party-api/v1/respondents/{party_id}/password-verification-token/{token}", headers=self.auth_headers
+        )
+        self.assertStatus(response, 200)
```

Comment:
Asserting on just 200 is not the best, we should be more explicit, same with the other asserts

Reply:
Added expected_status to give more freedom when testing